src/mdx-pages/blog/videojs-8-and-vhs-3.mdx

```diff
@@ -0,0 +1,163 @@
+
+### Using `loadMedia` to Populate the `TitleBar`
+
+The easiest way to provide a title and/or description is to use your player's `loadMedia` method:
+
+```js
+player.loadMedia({
+  artist: 'Disney',
+  album: 'Oceans',
+  title: 'Oceans',
+  description: 'Journey in to the depths ... and race with dolphins at play.',
+  poster: 'https://vjs.zencdn.net/v/oceans.png',
+  src: [{
+    src: 'https://vjs.zencdn.net/v/oceans.mp4',
+    type: 'video/mp4'
+  }, {
+    src: 'https://vjs.zencdn.net/v/oceans.webm',
+    type: 'video/webm'
+  }]
+})
+```
+
+If you're not familiar with the `loadMedia` method, it's a way to provide additional metadata for your media beyond what is available through the `src` method alone. As you see in the above example, the `title` and `description` are provided and will be used to populate the title bar.
+
+### Populating the `TitleBar` Directly
+
+The `TitleBar` can also be populated through direct input using the component's `update` method:
+
+```js
+player.titleBar.update({
+  title: 'Oceans',
+  description: 'Journey in to the depths ... and race with dolphins at play.'
+});
+```
+
+The title and/or description can be removed by passing an empty string for one or both of these values:
+
+```js
+player.titleBar.update({
+  title: '',
+  description: ''
+});
+```
+
+If both are removed, the `TitleBar` will no longer be visible.
+
+## New Video.js Utility Objects
+
+Over time, the `videojs` namespace has become cluttered with a wide variety of utility functions. One of the changes we are introducing is a more deliberately designed interface for these utility functions.
+
+The guiding principle here was that if a function didn't feel like a core part of the library, but was still _potentially useful_, we exposed it as part of a utility object instead of a top-level function.
+
+These are the utility objects attached to `videojs` in 8.0.0:
+
+Object                | Description
+----------------------|--------------------------------------
+`videojs.browser`     | Various user-agent detection values (available previously)
+`videojs.dom`         | DOM functions (available previously)
+`videojs.fn`          | Function... functions
+`videojs.num`         | Number functions
+`videojs.obj`         | Object functions
+`videojs.str`         | String functions
+`videojs.time`        | Time-related functions
+`videojs.url`         | URL-related functions
+
+<div class="note">
+  <p><b>NOTE:</b> For details on migrating from Video.js 7.x to 8.x, please see <a href="/guides/videojs-7-to-8/">our migration guide</a>!</p>
+</div>
+
+## Conclusion
+
+Thanks again to all our contributors and to the large community of developers who support Video.js by using it every day!
+
+If you'd like to contribute to Video.js, the place to start is our [CONTRIBUTING](https://github.com/videojs/video.js/blob/main/CONTRIBUTING.md) guide on GitHub.
```

Comment:
Do we still have `great-first-issue` issues? If we do this would be a good place to add it. Maybe it's worth adding this blog post to the README for awhile? We should probably link to the blog on github anyway...